proposals/0303-swiftpm-extensible-build-tools.md

```diff
@@ -0,0 +1,890 @@
+# Package Manager Extensible Build Tools
+
+* Proposal: [SE-0303](0303-filename.md)
```

Comment:
The link is broken here. `filename` should be replaced with `swiftpm-extensible-build-tools`.

Reply:
good catch, it was fixed in https://github.com/apple/swift-evolution/commit/4a4ff09ad223c6f1570d9531480974f1afbc5f70